src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
> It would be stupid to leave this method in though, since it wouldn't even get called anymore since class preloader 3.x.

Yeah, but the deps constraint is still `"classpreloader/classpreloader": "~2.0|~3.0",`, not `"~3.0"`. If you want to remove it on the 5.2.0 release then fine, it a major version. But please becareful on patch release.

Does symfony remove `@internal` method in patch release?